runtime/integration-tests/src/utils/accounts.rs

```diff
@@ -249,65 +130,6 @@ impl std::str::FromStr for Keyring {
 }
 
 /// Returns a Vector of default accounts
-///
-/// Accounts:
-/// * Keyring::Admin
-/// * Keyring::Alice
-/// * Keyring::Bob
-/// * Keyring::Ferdie
-/// * Keyring::Charlie
-/// * Keyring::Dave
-/// * Keyring::Eve
-/// * Keyring::TrancheInvestor(1)
```

Comment:
I actually liked the comment. Of course we dont need a line for every single tranche investor but do you think we could bring back something like Admin, ... Eve, TranchInvestor(1), ..., TrancheInvestor(50)?

Reply:
Sure, but I'm just curious, what add the comment that does not tell you the own implementation?

Reply:
Fair point. I often leverage rustdocs from hovering over. In this case, it is crystal clear so lets keep as is!